Week2 Pull request with new function

Lorry7 opened the pull request:
Includes new function get_GUI to call the shiny app and all updated descriptions.

Lorry7 committed:
Week2 Pull request with new function

Includes new function get_GUI to call the shiny app and all updated descriptions.

elena-sek commented on RocketsNEW/R/Functions_Laureen.R:
I was trying to run your app but I got the following error every time I tried (I tried it about 10 times). I saw you mentioned it in the note of the getRocketData function so maybe it was just my internet connection/laptop that I couldn't load it.

Error in curl::curl_fetch_memory(url, handle = handle) : 
  Timeout was reached: [lldev.thespacedevs.com] SSL/TLS connection timeout

elena-sek reviewed (commented):
I wasn't able to load the app but other than that the code has a logical flow and I didn't find that any parts woud be missing.

jessiwitti commented:
@elena-sek I had the same problem in the beginning! You need to load the curl package manually with library(curl). I've also loaded shiny and dplyr to be sure. I think @Lorry7 you need to add the curl package to your imports in the description file.
+ I like that you mention this error in your helper file haha 

Now it works for me and it looks really nice! I like that you can zoom in and click on the points to get more information. 
A small thing: I haven't clicked through every rocket launch, but in the Simorgh | Mahda, Keyhan-2 & Hatef-1 the picture does not load

Regarding the documentation: I see in your files that you have the help page included, but everytime I try to run ?get_GUI() or the other function in r, it shows me that there are no results for this. I don't know whether this is just me or doesn't work at all.

Lorry7 committed:
Updates to package

Added a part to retrieve newer data from  the Launch API, changed some descriptions for the function and added the curl package to imports. Furthermore, I added the libraries in front of the functions in get_GUI.
